setup.py

```diff
@@ -23,7 +23,8 @@
     'win-inet-pton;python_version<"3.0" and platform_system=="Windows"',
     # shutil.get_terminal_size and which were added in Python 3.3
     'backports.shutil_which;python_version<"3.3"',
-    'backports.shutil_get_terminal_size;python_version<"3.3"'
+    'backports.shutil_get_terminal_size;python_version<"3.3"',
+    'esprima'
```

Comment:
We should pin to this specific version, or possible a smaller range of versions. 

```suggestion
    'esprima==4.0.1'
```